.gitlab-ci.yml

```diff
@@ -46,5 +46,6 @@ test:smoke-build:
       -DCONFIG_MENDER_CLIENT_ADD_ON_TROUBLESHOOT=OFF
       -DCONFIG_MENDER_FULL_PARSE_ARTIFACT=ON
       -DCONFIG_MENDER_PROVIDES_DEPENDS=ON
+      -DCONFIG_MENDER_ALL_WARNINGS_AS_ERRORS=ON
```

Comment:
Do we have the script to run the above yet? It should be in some place with a proper name and available for people to run this when testing things locally.

Reply:
This think @danielskinstad is working on that here https://github.com/mendersoftware/mender-mcu/pull/30